sycl/test-e2e/Basic/vector/load_store.cpp

```diff
@@ -0,0 +1,187 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// RUN: %if preview-breaking-changes-supported %{ %{build} -fpreview-breaking-changes -o %t2.out %}
+// RUN: %if preview-breaking-changes-supported %{ %{run} %t2.out %}
+
```

Comment:
Can we extend this test case for `bfloat16` as well?

Reply:
The test has been extended to cover more types, vector of `bfloat16` included.